Assets/MixedRealityToolkit/Inspectors/Utilities/HandednessInspectorGUI.cs

```diff
@@ -0,0 +1,29 @@
+﻿
```

Comment:
Even though it's technically not needed any more, for consistency please add a copyright block as described in [coding guidelines here](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#script-license-information-headers)

Reply:
I stand corrected!

https://docs.opensource.microsoft.com/content/releasing/copyright-headers.html#copyright-headers-in-microsoft-oss-project

"Microsoft should not add a Microsoft copyright header to third party contributions to Microsoft OSS projects. We should never remove copyright notices from files."

No need to add this.